### PR TITLE
Stopwatch

### PR DESCRIPTION
**Pull request by apu52**
### Describe the project you want to add with tech stack

Hello Sir👨‍💻 ,I am Arpan Chowdhury, a aspiring Web developer.🤖 Here is the project of the front-end Web Development. We all use the most famous Application in our phone named STOPWATCH

### **Expected behavior**
I have made it using JS & CSS (mainly) .

### Video
https://github.com/apu52/Stopwatch-maker/assets/114172928/2cc065fe-f0bd-4759-9fa9-71d50844f31c

### Additional context (optional )

Please take a review of my PR & merge it under HACTOBERFEST 2023 .

### CODE REVIEW

**Comment by apu52** [comment]
@sadanandpai please take a look in my PR & after reviewing , Please merge it under HACTOBERFERST 2023.

**Comment by apu52** [comment]
@sadanandpai please take a look in my PR & after reviewing , Please merge it under HACTOBERFERST 2023.

**Comment by apu52** [comment]
@sadanandpai please take a look in my PR & after reviewing , Please merge it under HACTOBERFERST 2023.

File: native/machine-coding/Stopwatch/Contributors.md
Comment (sadanandpai) [inline native/machine-coding/Stopwatch/Contributors.md:1]: why is this needed

File: native/machine-coding/Stopwatch/Contributing.md
Comment (sadanandpai) [inline native/machine-coding/Stopwatch/Contributing.md:1]: why is this needed

File: native/machine-coding/Stopwatch/style.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #8c52ff;
+}
```
Comment (sadanandpai) [inline native/machine-coding/Stopwatch/style.css:14]: please import the core css as done for other challenges which comes with required styles and avoid modifying HTML and Body styles

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by apu52** [comment]
@sadanandpai I have resolved the merge conflicts in Style.css file. Can you please check It??
& merge the PR

**Comment by apu52** [comment]
@sadanandpai I have resolved the merge conflicts in Style.css file. Can you please check It??
& merge the PR

**Comment by apu52** [comment]
@sadanandpai I have resolved the merge conflicts in Style.css file. Can you please check It??
& merge the PR

**Comment by apu52** [comment]
I have deleted those files & have changed the codes . @sadanandpai Can you please take a review & please merge it

File: native/helpers/challenges.js [hunk 2]
```diff
@@ -426,6 +426,24 @@ export const challenges = [
...
+    difficulty: 'moderate',
+    developer: 'apu52',
+    tags: [],
+
+    title: 'AI Chess Game',
+    link: 'AI-CHESS-Game',
+    difficulty: 'hard',
+    developer: 'AnkitaSikdar005',
+    tags: [],
+    isNew: true,
```
Comment (sadanandpai) [inline native/helpers/challenges.js:443]: why do we need AI chess game details for stop watch challenge

File: native/machine-coding/Stopwatch/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Stopwatch</title>
+    <link rel="stylesheet" href="style.css" />
```
Comment (sadanandpai) [inline native/machine-coding/Stopwatch/index.html:8]: please add the required tags as done for other challenges

File: native/machine-coding/Stopwatch/script.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
+
+let [milliseconds, seconds, minutes, hours] = [0, 0, 0, 0];
+let timeRef = document.querySelector(".timer-display");
+let laps = document.querySelector("#laps-timer");
+let lapsItems = document.querySelector(".laps-items");
```
Comment (sadanandpai) [inline native/machine-coding/Stopwatch/script.js:5]: use const for not reassigned variables

File: native/machine-coding/Stopwatch/style.css [hunk 5]
```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap'); 
+
+@import url('https://fonts.googleapis.com/css2?family=Aclonica&display=swap');
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```
Comment (sadanandpai) [inline native/machine-coding/Stopwatch/style.css:10]: import the core styles. 
And do not add new fonts.
No need to add * rule set

**Review by sadanandpai — CHANGES_REQUESTED** [review]

File: native/machine-coding/Stopwatch/index.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        <button id="pause-timer">Pause</button>
+        <button id="reset-timer">Reset</button>
+      </div>
+     <div class="laps-container">
+      <button id="laps-timer">Laps</button>
+     </div>
+    </div>
+    <div class="laps-items">
+     </div>
+    <script src="script.js"></script>
```
Comment (sadanandpai) [inline native/machine-coding/Stopwatch/index.html:26]: move script to head with defer

File: native/machine-coding/Stopwatch/style.css [hunk 7]
```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap'); 
+
+@import url('https://fonts.googleapis.com/css2?family=Aclonica&display=swap');
```
Comment (sadanandpai) [inline native/machine-coding/Stopwatch/style.css:3]: do not add new fonts

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Pull request closed without merging**